source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -29,10 +29,13 @@ by calling the ``find()`` or ``findOne()`` method.
 
 .. tip:: Interactive Lab
    
-   You can complete a short interactive lesson that demonstrates how to
-   retrieve data by using the ``find()`` method in an in-browser coding
-   experience. To learn more, see the :ref:`node-retrieve-instruqt-lab`
-   section of this guide.
+   This page includes a short interactive lesson that demonstrates how to
+   retrieve data by using the ``find()`` method. You can complete this lab
```

Comment:
S: Since we're referring to it initially as a lesson
```suggestion
   retrieve data by using the ``find()`` method. You can complete this lesson
```

Reply:
ended up changing all mentions of "lesson" to "lab," since I think Instruqt is better described as a lab, but fixed!